WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagsViewController.swift

```diff
@@ -144,14 +143,14 @@ final class SiteTagsViewController: UITableViewController {
         }
     }
 
-    private func setupNavigationBar() {
-        configureRightButton()
-    }
-
     private func configureRightButton() {
-        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add,
-                                                            target: self,
-                                                            action: #selector(createTag))
+        if let count = resultsController.fetchedObjects?.count, count > 0 {
```

Comment:
One observation : this line is almost identical to the caller of this method (`refreshNoResultsView()`).

It might be slightly more DRY to remove that conditional, and modify the signature of `configureRightButton` to accept a `Bool` parameter (e.g., `enabled`) that influenced whether the `rightBarButtonItem` is set or not.

I don't feel strongly about it, but wondered if you think that's a worthwhile change?